execution_engine/src/core/engine_state/step.rs

```diff
@@ -129,16 +129,28 @@ pub enum StepError {
     TypeMismatch(StoredValueTypeMismatch),
     #[error("Era validators missing: {0}")]
     EraValidatorsMissing(EraId),
-    #[error(transparent)]
```

Comment:
Hmm, why change this? The type still derives `thiserror` (see `#[derive(Error)]` above, so `#[from]` and `#[transparent]` still work. I believe they also set source correctly.